src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  rockets: [],
```

Comment:
[Optional] I kindly suggest if you provide an initial rocket data rather than an empty array